package/carbonio-clamav-signature-provider.hcl

```diff
@@ -0,0 +1,17 @@
+services {
+  checks = [
+    {
+      http     = "http://127.0.0.1:10000",
```

Comment:
The service should be exposed on a different IP, not localhost, as per service guidelines. Something like 127.78.0.X

Reply:
done :+1: